packages/datadog-plugin-couchbase/src/index.js

```diff
@@ -74,7 +73,9 @@ function createWrapN1qlReq (tracer, config) {
 
       const scope = tracer.scope()
       const n1qlQuery = q && q.statement
-      const span = startQuerySpan(tracer, config, n1qlQuery)
+      const span = startSpan(tracer, config, 'query', n1qlQuery)
+
+      span.setTag('span.type', 'sql')
```

Comment:
This is the generic type for databases that use SQL or similar. It was also already used before this change, but it was on all spans since only N1QL queries were supported.